src/app/components/layout/header/header.component.ts

```diff
@@ -25,5 +32,16 @@ export class HeaderComponent implements OnInit {
       } else {
           disableScroll.off();
       }
+      this.setupMenu();
```

Comment:
I think this was done to hide all the submenus when opening the menu in a small screen. However, this is not working, since when modifying the `open` property in `onCLickMenuDropdown` the property is modified in both `this.menu` and the original `menu` object, because `this.menu` is simply a reference, not a copy.